### PR TITLE
Adjust documentation updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Adjust documentation updates

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Adjust app token FAQ](https://www.notion.so/rudderstacks/Adjust-app-token-FAQ-bbc30c5131264bde90a783f346b34a66)
> [Adjust: Supporting partner params](https://www.notion.so/rudderstacks/Adjust-supporting-partner-params-6a43689dd5884b8f97e71c904eec65c6)

### CODE REVIEW

File: docs/destinations/streaming-destinations/adjust.mdx [hunk 1]
```diff
@@ -256,22 +291,46 @@ Here is an example:
...
+<div class="infoBlock">
+For more information on adding a partner in Adjust, refer to the <Link to="#how-can-i-set-up-new-partners-in-adjust">FAQ</Link> section below.
+</div>
+
+<div class="warningBlock">
+Make sure you have defined the property mapping in the <strong>RudderStack Parameters to Partner Parameters</strong> dashboard setting. Adjust will reject any properties apart from these mappings.
+</div>
+
+RudderStack uses the property mappings specified in the **RudderStack Parameters to Partner Parameters** setting to check if a key is present in the `track` event properties and maps it to the corresponding Adjust partner parameter object.
+
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/adjust.mdx:330]: @ItsSudip added this as a note.

File: docs/destinations/streaming-destinations/adjust.mdx [hunk 2]
```diff
@@ -256,22 +291,80 @@ Here is an example:
...
+  "gps_adid": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+  "att_status": 3,
+  "tracking_enabled": true,
+  "currency": "USD",
+  "ip_address": "[::1]",
+  "s2s": 1,
+  "app_token": "t1yurrb968zk",
+  "event_token": "tf4gm5",
+  "environment": "production",
+  "partner_params": "{\"price\":20.99,\"quantity\":10}"
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/adjust.mdx]: Added this as a note below the example.

File: docs/destinations/streaming-destinations/adjust.mdx [hunk 3]
```diff
@@ -280,23 +381,85 @@ For `revenue` event, add the fields `total` and `currency` to `properties` and i
...
 <div class="warningBlock">
-Install Attribution is not supported in <code class="inline-code">cloud-mode</code>
+This feature is not supported when sending events via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
+</div>
+
+<div class="infoBlock">
+For testing install attribution and troubleshooting tips, refer to the <a href="https://help.adjust.com/en/article/testing-console">Adjust documentation</a>.
+</div>
+
+## Adjust environment and logging
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/adjust.mdx:404]:
```suggestion
## Adjust environment dependency on log level 
```

File: docs/destinations/streaming-destinations/adjust.mdx [hunk 4]
```diff
@@ -280,23 +381,85 @@ For `revenue` event, add the fields `total` and `currency` to `properties` and i
...
+<div class="infoBlock">
+For more information on the <code class="inline-code">RudderLogLevel</code>, refer to the following SDK documentation:
+<ul>
+  <li><a href="/sources/sdks/rudderstack-android-sdk/#configuring-your-rudderstack-client">Android</a></li>
+  <li><a href="/sources/sdks/rudderstack-ios-sdk/#configuring-the-rudderstack-client">iOS</a></li>
+</ul>
 </div>
 
-## Environment and logging
+`RudderLogLevel.DEBUG` or higher will set the Adjust environment to **Sandbox** and `logLevel` to `VERBOSE`. For anything below that, RudderStack sets the Adjust environment to **Production** and `logLevel` to `ERROR`.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/adjust.mdx:416]: |  RudderLogLevel | Adjust environment      | loglevel   | 
| :------------------ | :------------ | :------------ | 
| Debug or higher     |     Sandbox          |       verbose        | 
| lower     |     Production          |     ERROR          |        

Can check what are the allowed values for RudderLogLevel apart from Debug

File: docs/destinations/streaming-destinations/adjust.mdx [hunk 5]
```diff
@@ -280,23 +381,85 @@ For `revenue` event, add the fields `total` and `currency` to `properties` and i
...
+To get your Adjust app token, follow these steps:
+
+1. Log into your [Adjust dashboard](https://dash.adjust.com/#/).
+2. Find your app and select the app options caret (^), as shown:
+
+<img src="../../assets/event-stream-destinations/adjust-app-token-1.png" alt="Adjust app token" />
+
+3. You will find your app token listed here.
+
+<img src="../../assets/event-stream-destinations/adjust-connection-settings.png" alt="Adjust app token"/>
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/adjust.mdx:431]:
```suggestion
<img src="../../assets/event-stream-destinations/adjust-app-token-2.png" alt="Adjust app token"/>
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**